_source/logzio_collections/_tracing-sources/cs_open_tracing.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Sending traces from C# application
+logo:
+  logofile: dotnet.svg
+  orientation: vertical
+data-source: dotnet
+description: How to send traces from C# application
+open-source:
+  - title: 
+    github-repo: jaeger-logzio
+contributors:
+  - yyyogev
+  - yberlinger
+shipping-tags:
+  - from-your-code
+  - traces
+---
+<!-- tabContainer:start -->
+<div class="branching-container">
+
+* [Auto instrumentation](#tab1)
+* [From your code](#tab2)
+{:.branching-tabs}
+
+<!-- tab:start -->
+<div id="tab1">
+
+# OpenTracing Instrumentation for C\#
+
+This article describes how to automatically extract Traces from your C# based app with OpenTracing instrumentation.
+
+## OpenTracing and Jaeger
+
+OpenTracing is an API standard of instrumentation for distributed tracing.
+Jaeger is one implementation of that standard and our most recommended practice for C#. It has a set of libraries and tools to provide open source implementation 
+for distributed tracing.
+
+## OpenTracing for C\#
+
+### Requirements
+Add the following packages to your solution:
+
+* OpenTracing.Contrib.NetCore
+
+* OpenTracing
+
+* Jaeger
+
+
+### Implementation
+Add to you `Stratup.cs` file the following code snippet:
```

Comment:
where is the file located? Do we need to provide navigation guidance for this?

Reply:
Yes, should be capitalized.
No need, it's basic and it's in the root.